observability/WFLY-20296-add-support-for-more-opentelemetry-config-options.adoc

```diff
@@ -0,0 +1,140 @@
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+While every aspect of OpenTelemetry is technically configurable via MP Telemetry and MP Config, there have been several requests for more direct support for the various options in the management model. The RFE will identify and implement the options most likely to be of general interest to WildFly users.
+
+=== User Stories
+
+- As an application developer, I would like greater control in how OpenTelemetry is configured in the server without having to add an external library (e.g., MicroProfile Config) to my deployment(s).
+
+== Issue Metadata
+
+- https://issues.redhat.com/browse/WFLY-20296[]
+
+=== Related Issues
+
+- https://issues.redhat.com/browse/WFLY-18290[]
+- https://issues.redhat.com/browse/WFLY-19919[]
+- https://issues.redhat.com/browse/JBEAP-27199[]
+
+=== Affected Projects or Components
+
+This change will affect only the OpenTelemetry subsystem
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
+
+* Traditional standalone server (unzipped or provisioned by Galleon)
+* Managed domain
+* OpenShift Source-to-Image (S2I)
+* Bootable jar
+
+== Requirements
+
+* Additional configuration options should include:
+** General
+*** TLS server certificate (`otel.exporter.otlp.certificate`)
+*** TLS client certificate (`otel.exporter.otlp.client.certificate`)
+*** Client key (`otel.exporter.otlp.client.key`)
+*** Compression (`otel.exporter.otlp.compression`)
+** Metrics
+*** Export interval (`otel.metric.export.interval`)
+** Logs
+*** Schedule delay (`otel.blrp.schedule.delay`)
+*** Max queue size (`otel.blrp.max.queue.size`)
+*** Max batch size (`otel.blrp.max.export.batch.size`)
+*** Export timeout (`otel.blrp.export.timeout`)
+* Changes to the management model should be split out based on signal types
+** For ease of use by users, the structure of the updated XML should reflect this
+** Existing trace-specific configuration options should be moved to the trace child element
+** The shape of the management model need not share the nested nature unless that is a more natural structure in the Java code
+** Shared properties (e.g., the OpenTelemetry Collector endpoint) should be exposed on the root element.
+*** Signal-specific overrides need not be supported
+* All new properties added will have, as defaults, values described in the relevant OpenTelemetry documentation.
+* Support should not depend on, e.g., MicroProfile Telemetry or MicroProfile Config
+* Any configured properties should still be able to be overridden via the existing MicroProfile support
+* While it is acceptable to relate WildFly management model attributes to their respective system property in the OpenTelemetry documentation to help explain their intent, the system property is an implementation detail that should not necessarily be leaked in the model to avoid as much coupling as is possible.
```

Comment:
As work on this progresses, please tighten this language. It's fine now for initial exploration but by the end it should be stated as a requirement, i.e. what must/must not be done.